tests/integration/test_mysql_protocol/java_client_text.reference

```diff
@@ -0,0 +1,129 @@
+### testSimpleDataTypes
+Row #1
+i8 type is TINYINT, value: -128
+i16 type is SMALLINT, value: -32768
+i32 type is INT, value: -2147483648
+i64 type is BIGINT, value: -9223372036854775808
+i128 type is CHAR, value: -170141183460469231731687303715884105728
+i256 type is CHAR, value: -57896044618658097711785492504343953926634992332820282019728792003956564819968
+ui8 type is TINYINT, value: 120
+ui16 type is SMALLINT, value: 1234
+ui32 type is INT, value: 51234
+ui64 type is BIGINT, value: 421342
+ui128 type is CHAR, value: 15324355
+ui256 type is CHAR, value: 41345135123432
+f32 type is FLOAT, value: -0.796896
+f64 type is DOUBLE, value: -0.113259
+b type is TINYINT, value: true
+Row #2
+i8 type is TINYINT, value: 127
+i16 type is SMALLINT, value: 32767
+i32 type is INT, value: 2147483647
+i64 type is BIGINT, value: 9223372036854775807
+i128 type is CHAR, value: 170141183460469231731687303715884105727
+i256 type is CHAR, value: 57896044618658097711785492504343953926634992332820282019728792003956564819967
+ui8 type is TINYINT, value: 255
+ui16 type is SMALLINT, value: 65535
+ui32 type is INT, value: 4294967295
+ui64 type is BIGINT, value: 18446744073709551615
+ui128 type is CHAR, value: 340282366920938463463374607431768211455
+ui256 type is CHAR, value: 115792089237316195423570985008687907853269984665640564039457584007913129639935
+f32 type is FLOAT, value: 1.234000
+f64 type is DOUBLE, value: 3.352451
+b type is TINYINT, value: false
+
+### testStringTypes
+Row #1
+s type is CHAR, value: 42
+sn type is CHAR, value: null
+lc type is CHAR, value: test
+nlc type is CHAR, value: null
```

Comment:
This entire reference was copy-pasted only because of how NULLs are reported via binary (`ᴺᵁᴸᴸ`) vs text (`null`). We need to double-check whether our text protocol implementation follows MySQL standard, and maybe this entire copy-pasted reference will be removed as a follow-up.